installer/manifests/keptn/charts/control-plane/templates/core.yaml

```diff
@@ -281,7 +281,7 @@ spec:
                 fieldRef:
                   fieldPath: metadata.namespace
             - name: MONGODB_HOST
-              value: 'mongodb:27017'
+              value: '{{ .Values.mongodb.host }}:{{ .Values.mongodb.port }}'
```

Comment:
Please also use the templating in the `configuration-service` deployment, line 419.

Reply:
@agrimmer done, missed that one. thank you!